seed/seedwriter/writer.go

```diff
@@ -1240,15 +1269,32 @@ func (w *Writer) resolveValidationSetAssertion(seq *asserts.AtSequence) (asserts
 	return seq.Resolve(w.db.Find)
 }
 
-func (w *Writer) validationSets() (*snapasserts.ValidationSets, error) {
-	valsets := snapasserts.NewValidationSets()
+func (w *Writer) validationSetAsserts() (map[*asserts.AtSequence]*asserts.ValidationSet, error) {
+	vsasserts := make(map[*asserts.AtSequence]*asserts.ValidationSet)
```

Comment:
super nitpick/question: should this all be lowercase?

Reply:
You are right, it shouldn't be, fixed